### PR TITLE
Add CSS variable documentation

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: doc/integrator/ngeo.rst.tmpl [hunk 1]
```diff
@@ -117,3 +99,14 @@ You also have a ``default`` entry from where we can give the base values for ``c
...
 `ngeo constants <https://camptocamp.github.io/ngeo/${MAIN_BRANCH}/jsdoc/module-src_options.html>`_,
 `gmf constants <https://camptocamp.github.io/ngeo/${MAIN_BRANCH}/jsdoc/module-contribs_gmf_src_options.html>`_.
+
+CSS style
+---------
+
+In the ``gmfOptions`` we have a ``cssVars`` property where we can configure the CSS variable
+(``var(--a-variable)`` in the css files.
+
+On this dictionaty we have some particular variable named ``brand-primary`` and ``brand-secoundary``,
```
Comment (jwkaltz) [inline doc/integrator/ngeo.rst.tmpl:109]: dictionaty -> dictionary
variable -> variables
brand-secoundary -> brand-secondary

File: doc/integrator/ngeo.rst.tmpl [hunk 2]
```diff
@@ -117,3 +99,14 @@ You also have a ``default`` entry from where we can give the base values for ``c
...
+thoses variable confibure the major applications colors.
```
Comment (jwkaltz) [inline doc/integrator/ngeo.rst.tmpl:110]: confibure -> configure
applications -> application

Comment (sbrunner) [reply doc/integrator/ngeo.rst.tmpl]: Thanks :-)

File: doc/integrator/ngeo.rst.tmpl [hunk 3]
```diff
@@ -117,3 +99,14 @@ You also have a ``default`` entry from where we can give the base values for ``c
 The possible values are described in the ngeo documentation
 `ngeo constants <https://camptocamp.github.io/ngeo/${MAIN_BRANCH}/jsdoc/module-src_options.html>`_,
 `gmf constants <https://camptocamp.github.io/ngeo/${MAIN_BRANCH}/jsdoc/module-contribs_gmf_src_options.html>`_.
+
+CSS style
+---------
+
+In the ``gmfOptions`` we have a ``cssVars`` property where we can configure the CSS variable
```
Comment (jwkaltz) [inline doc/integrator/ngeo.rst.tmpl:106]: Which CSS variable?
Or do we mean here "all CSS variables"?

File: doc/integrator/ngeo.rst.tmpl [hunk 4]
```diff
@@ -117,3 +99,14 @@ You also have a ``default`` entry from where we can give the base values for ``c
...
+(``var(--a-variable)`` in the css files.
+
+On this dictionary we have some particular variables named ``brand-primary`` and ``brand-secondary``,
```
Comment (jwkaltz) [inline doc/integrator/ngeo.rst.tmpl:109]: "this dictionary" - this means the content of cssVars? That this is a dictionary?

File: doc/integrator/ngeo.rst.tmpl [hunk 5]
```diff
@@ -117,3 +99,14 @@ You also have a ``default`` entry from where we can give the base values for ``c
...
+thoses variable configure the major application colors.
+
+You also have an ``<interface>.css`` file to overide the css rules.
```
Comment (jwkaltz) [inline doc/integrator/ngeo.rst.tmpl:112]: Do we mean:
"You can in addition define an interface.css file to override the CSS rules"
?

File: doc/integrator/ngeo.rst.tmpl (hunk 4 above)
Comment (jwkaltz) [inline doc/integrator/ngeo.rst.tmpl:109]: This confuses me a bit... I suppose the dictionary is empty, but the integrator can add any variables she wishes (variables which correspond to the CSS variable names). And "brand-primary" and "brand-secondary" are then somehow not normal CSS variables used in GMF, but something hard-coded?

File: doc/integrator/ngeo.rst.tmpl [hunk 6]
```diff
@@ -117,3 +99,14 @@ You also have a ``default`` entry from where we can give the base values for ``c
...
 `gmf constants <https://camptocamp.github.io/ngeo/${MAIN_BRANCH}/jsdoc/module-contribs_gmf_src_options.html>`_.
+
+CSS style
+---------
+
+In the ``gmfOptions`` we have a ``cssVars`` property where we can configure the CSS variable
+(``var(--a-variable)`` in the css files.
+
+On this dictionary we have some particular variables named ``brand-primary`` and ``brand-secondary``,
+thoses variable configure the major application colors.
```
Comment (jwkaltz) [inline doc/integrator/ngeo.rst.tmpl:110]: thoses variable -> those variables

**Review by jwkaltz — APPROVED** [review]

File: doc/integrator/ngeo.rst.tmpl (hunk 3 above)
Comment (sbrunner) [reply doc/integrator/ngeo.rst.tmpl:106]: => all :-)

File: doc/integrator/ngeo.rst.tmpl (hunk 5 above)
Comment (sbrunner) [reply doc/integrator/ngeo.rst.tmpl:112]: +1

File: doc/integrator/ngeo.rst.tmpl (hunk 4 above)
Comment (sbrunner) [reply doc/integrator/ngeo.rst.tmpl]: I completely rephrase it, I hope that's clearer...

File: doc/integrator/ngeo.rst.tmpl [hunk 7]
```diff
@@ -117,3 +99,16 @@ You also have a ``default`` entry from where we can give the base values for ``c
...
 `ngeo constants <https://camptocamp.github.io/ngeo/${MAIN_BRANCH}/jsdoc/module-src_options.html>`_,
 `gmf constants <https://camptocamp.github.io/ngeo/${MAIN_BRANCH}/jsdoc/module-contribs_gmf_src_options.html>`_.
+
+CSS style
+---------
+
+In the ``gmfOptions`` we have a ``cssVars`` property where we can configure all CSS variable
+(``var(--a-variable)`` in the css files.
+
+The content of ``cssVars`` is a dictionary witch the keys are the CSS variable names, and the values the
```
Comment (jwkaltz) [inline doc/integrator/ngeo.rst.tmpl:109]: witch -> where

File: doc/integrator/ngeo.rst.tmpl [hunk 8]
```diff
@@ -117,3 +99,16 @@ You also have a ``default`` entry from where we can give the base values for ``c
...
+value to be set to the CSS variable.
+For the CSS variable named ``brand-primary`` and ``brand-secondary`` we also calculated the other colors
```
Comment (jwkaltz) [inline doc/integrator/ngeo.rst.tmpl:111]: variables

File: doc/integrator/ngeo.rst.tmpl [hunk 9]
```diff
@@ -117,3 +99,16 @@ You also have a ``default`` entry from where we can give the base values for ``c
...
+related to thoes variable as it's done in the SCSS, to be able to thnge the major application colors.
```
Comment (jwkaltz) [inline doc/integrator/ngeo.rst.tmpl:112]: thoes -> those
thnge -> change (I suppose)

**Review by jwkaltz — APPROVED** [review]

**Comment by sbrunner** [comment]
Thanks @jwkaltz :-)

**Pull request merged**